commit f5c771d07a863bb2d70bdfb39ffa938176da8756 (refs/changes/96/396/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-01T15:18:15-07:00 (3 years, 1 month ago)
    
    RELENG-715 ironically the "ENG" jira project is not an engineering project
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/tools/jira/listengprojects.sh b/tools/jira/listengprojects.sh
index b54aa8a..2ab1e3d 100755
--- a/tools/jira/listengprojects.sh
+++ b/tools/jira/listengprojects.sh
@@ -46,7 +46,6 @@ DATASET
 DCAPI
 DOC
 DOCKER
-ENG
 FWAPI
 HEAD
 HVM
